plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java

```diff
@@ -1141,6 +1142,7 @@ public static boolean isHudiTable(Table table)
         String inputFormat = table.getStorage().getStorageFormat().getInputFormatNullable();
 
         return HUDI_PARQUET_INPUT_FORMAT.equals(inputFormat) ||
+                HUDI_MAPRED_INPUT_FORMAT.equals(inputFormat) ||
```

Comment:
This is rather new to me. Separate commit pls.
How did you stumble upon it?

cc @codope did we miss this one while we agreed on the strategy for figuring out whether we're dealing with a Hudi table?

Reply:
it's a mistake actually